### PR TITLE
Made the README better (again)

### PR DESCRIPTION
**Pull request by AronWinski**

### CODE REVIEW

**Comment by AronWinski** [comment]
Check out my major improvements!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
 # fall-2020-r1
 Main repository for the first Unity project of Fall 2020
+Testing changes
+
+I added another line, is that better?
+
+Oh, forgot to add this line!
```
Comment (nsk0007) [inline README.md:7]: This line is **impeccable**! You've really outdone yourself!

**Comment by AronWinski** [comment]
Just added a third line!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,7 @@
 # fall-2020-r1
 Main repository for the first Unity project of Fall 2020
+Testing changes
+
+I added another line, is that better?
```
Comment (nsk0007) [inline README.md:5]: GOOD LINE!!

**Review by nsk0007 — APPROVED** [review]
First line seemed sus, but the second and third cleared things up

File: README.md (hunk 2 above)
Comment (AronWinski) [reply README.md:5]: THANK YOU, THANK YOU! I worked so hard on it.

Comment (AronWinski) [reply README.md:5]: This really isn't quite as bad as I thought. With the review rule, we don't have to worry about people accidentally merging into master

Comment (nsk0007) [reply README.md:5]: But can you still push to master? Directly?

Comment (AronWinski) [reply README.md:5]: Nope!

**Comment by AronWinski** [comment]
Merging is totally blocked

**Review by jacob-thrasher — CHANGES_REQUESTED** [review]
Delete the weird files I think?

**Review by jacob-thrasher — APPROVED** [review]
This is amazing

**Pull request merged**